php/libraries/File_Upload.class.inc

```diff
@@ -508,144 +465,6 @@ class File_Upload
         return $new_name;
     }
 
-     /**
```

Comment:
removing the following functions seems to be removing some renaming options

@driusan @MounaSafiHarab either of you know why they were added in the first place ?? seems like imaging upploader is the only module that uses this class